packages/server/package.json

```diff
@@ -77,11 +76,9 @@
     "getos": "3.1.1",
     "glob": "7.1.3",
     "graceful-fs": "4.2.0",
-    "http-accept": "0.1.6",
```

Comment:
Seems this dep was completely removed. Can you explain why this was necessary?

Reply:
It's an unstable, pre-release package that's not on GitHub, and we were only using 1 function from it which was replaced with just `String.prototype.contains`